src/app/core/components/left-nav-tree/left-nav-tree.component.html

```diff
@@ -31,7 +31,7 @@
           <span class="mask-branch"></span>
         </span>
       </span>
-      <span class="node-item-content" [ngClass]="{ unlocked: !node.locked }" (click)="selectNode(node)">
+      <span class="node-item-content" [ngClass]="{ unlocked: !node.data.element.locked }" (click)="selectNode(node)">
```

Comment:
What is "unlocked"? Some are locked but the other one should be "normal", I don't see why a need for a specific unlocked class.

Reply:
Fixed